packages/messaging/tsconfig.json

```diff
@@ -1,8 +1,7 @@
 {
   "extends": "../../config/tsconfig.base.json",
   "compilerOptions": {
-    "outDir": "dist",
-    "strict": true
```

Comment:
Might make sense to create an issue to add this back later if possible. Admin Node SDK has had one for a while.